readme.md

```diff
@@ -16,7 +16,7 @@ Or, `F1` → `Install Extension` → Search for `ava`
 
 ## Snippets
 
-Included are some [snippets](snippets/ava.json) useful for writing AVA tests.
+Included are some [AVA](snippets/ava.json) and [assertion](snippets/assertions.json) useful for writing AVA tests.
```

Comment:
Couldn't come up with a name for the hook snippet stuff :). But now I see I forgot the word `snippet` as well.

> Included are some [AVA](snippets/ava.json) and [assertion](snippets/assertions.json) snippets useful for writing AVA tests.